cli/interpreter.cc

```diff
@@ -7,7 +7,7 @@
 int interpret_input(std::istream& is)
 {
 	try {
-		Mata::Parser::Parsed parsed = Mata::Parser::parse_vtf(is, true);
+		Mata::Parser::Parsed parsed = Mata::Parser::parse_mf(is, true);
```

Comment:
Parse motherfucker?

Reply:
Mata Format - mf

Reply:
I like parse motherfucker better. For me formula.mf will be "formula motherfucker"

Reply:
I think that the naming is not ideal, but I do not have problems with it either. I would keep the current name.